shared/src/test/scala/singleton/ops/ImplicitConversionSpec.scala

```diff
@@ -11,5 +11,5 @@ class ImplicitConversionSpec extends Properties("ImplicitConversion") {
   property("Wrong OpToOp") = {illTyped("""val ret : 2 + 1 = implicitly[4 - 2]"""); true}
   property("Wrong OpToSingleton") = {illTyped("""val ret : 3 = implicitly[4 - 2]"""); true}
   property("Wrong SingletonToOp") = {illTyped("""val ret : 2 + 1 = 4"""); true}
-  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueOf[4 - 1]}
-}
\ No newline at end of file
+  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueof.valueOf[4 - 1]}
```

Comment:
LBS is Lightbend Scala (the main scala branch). It too does not have `valueOf`, so I removed the dependency in #33 . You can choose to remove `ValueOf` completely from the code and tests, or I will remove it (and the scala-js workaround) after this PR is merged. Do as you wish. After #33, it will be gone.